doc/nrf/protocols/wifi/stack_configuration.rst

```diff
@@ -0,0 +1,298 @@
+       If the application does not need to send large frames, then this can be reduced to save memory.
+   * - :kconfig:option:`CONFIG_NRF700X_RX_MAX_DATA_SIZE`
+     - ``64`` to ``1600``
+     - Maximum RX data size
+     - Memory savings
+     - This controls the maximum size of the frames that can be received by the Wi-Fi protocol.
+       Large frame sizes imply more memory usage but can efficiently utilize the bandwidth.
+       If the application does not need to receive large frames, then this can be reduced to save memory.
+
+The configuration options must be used in conjunction with the Zephyr networking stack configuration options to achieve the desired performance and memory usage.
+These options form a staged pipeline all the way to the nRF70 Series chip, any change in one stage of the pipeline will impact the performance and memory usage of the next stage.
+For example, solving bottleneck in one stage of the pipeline might lead to a bottleneck in the next stage.
+
+.. _constrained_host_packet_memory:
+
+nRF70 Series packet memory
+**************************
+The nRF70 Series device chipset has a special memory called the packet memory to store the Wi-Fi protocol frames for both TX and RX.
+The various configuration options that control the size of the packet memory are listed below:
+
+* :kconfig:option:`CONFIG_NRF700X_TX_MAX_DATA_SIZE`
+* :kconfig:option:`CONFIG_NRF700X_RX_MAX_DATA_SIZE`
+* :kconfig:option:`CONFIG_NRF700X_MAX_TX_TOKENS`
+* :kconfig:option:`CONFIG_NRF700X_MAX_TX_AGGREGATION`
+* :kconfig:option:`CONFIG_NRF700X_RX_NUM_BUFS`
+
+The packet memory is divided into two parts, one for TX and one for RX. The size of the TX packet memory is calculated as follows:
+
+.. code-block:: none
+
+   (CONFIG_NRF700X_TX_MAX_DATA_SIZE + 52 ) * CONFIG_NRF700X_MAX_TX_TOKENS * CONFIG_NRF700X_MAX_TX_AGGREGATION
+
+The size of the RX packet memory is calculated as follows:
+
+.. code-block:: none
+
+   CONFIG_NRF700X_RX_MAX_DATA_SIZE * CONFIG_NRF700X_RX_NUM_BUFS
+
+The total packet memory size is calculated as follows:
+
+.. code-block:: none
+
+   (CONFIG_NRF700X_TX_MAX_DATA_SIZE + 52 ) * CONFIG_NRF700X_MAX_TX_TOKENS * CONFIG_NRF700X_MAX_TX_AGGREGATION +
+   CONFIG_NRF700X_RX_MAX_DATA_SIZE * CONFIG_NRF700X_RX_NUM_BUFS
+
+There is a build time check to ensure that the total packet memory size does not exceed the available packet memory size in the nRF70 Series chip.
+
+.. note::
+   The ``52`` bytes in the above equations are the overhead bytes required by the nRF70 Series chip to store the headers and footers of the Wi-Fi protocol frames.
+
+.. _constrained_host_usage_profiles:
+
+Usage profiles
+**************
+
+The nRF70 Series driver can be used in the following profiles (not an exhaustive list):
+
+.. list-table::
+   :header-rows: 1
+
+   * - Features
+     - Profile
+     - Configuration Options
+     - Use cases
+     - Throughputs
+   * - STA scan only
```

Comment:
This needs an :abbr: as well?

Reply:
Actually, the abbr is used where only STA mode is emphasized. Here, its a combination of STA and scan, hence left it as is.